components/CtaDownload/src/index.scss

```diff
@@ -1,24 +1,24 @@
 .denhaag-cta-download {
-  --denhaag-cta-download-icon-width-fallback: calc(var(--denhaag-cta-download-gap) * 4);
+  --denhaag-cta-download-icon-width-fallback: calc(var(--denhaag-cta-download-gap, var(--denhaag-space-lg)) * 4);
 
   align-items: center;
-  border-left: 1px solid var(--denhaag-cta-download-border-color, currentColor);
-  border-right: 1px solid var(--denhaag-cta-download-border-color, currentColor);
-  box-shadow: 0 1px 0 var(--denhaag-cta-download-border-color, currentColor),
-    inset 0 1px 0 var(--denhaag-cta-download-border-color, currentColor);
-  color: var(--denhaag-cta-download-color);
+  border-left: 1px solid var(--denhaag-cta-download-border-color, var(--denhaag-color-grey-2));
+  border-right: 1px solid var(--denhaag-cta-download-border-color, var(--denhaag-color-grey-2));
+  box-shadow: 0 1px 0 var(--denhaag-cta-download-border-color, var(--denhaag-color-grey-2)),
+    inset 0 1px 0 var(--denhaag-cta-download-border-color, var(--denhaag-color-grey-2));
+  color: var(--denhaag-cta-download-color, var(--denhaag-color-grey-4));
   display: flex;
   flex-direction: row;
-  font-family: var(--denhaag-cta-download-font-family, inherit);
-  font-size: var(--denhaag-cta-download-font-size, 1rem);
-  font-weight: var(--denhaag-cta-download-font-weight, inherit);
-  line-height: var(--denhaag-cta-download-line-height, inherit);
-  gap: var(--denhaag-cta-download-gap);
+  font-family: var(--denhaag-cta-download-font-family, var(--denhaag-typography-sans-serif-font-family));
+  font-size: var(--denhaag-cta-download-font-size, var(--denhaag-typography-scale-lg-font-size));
+  font-weight: var(--denhaag-cta-download-font-weight, var(--denhaag-typography-weight-regular));
+  line-height: var(--denhaag-cta-download-line-height, 1.3);
+  gap: var(--denhaag-cta-download-gap, var(--denhaag-space-lg));
   overflow-wrap: break-word;
-  padding-block-start: var(--denhaag-cta-download-padding-block, var(--denhaag-cta-download-gap));
-  padding-block-end: var(--denhaag-cta-download-padding-block, var(--denhaag-cta-download-gap));
-  padding-inline-start: var(--denhaag-cta-download-padding-inline, calc(var(--denhaag-cta-download-gap) / 1.5));
-  padding-inline-end: var(--denhaag-cta-download-padding-inline, calc(var(--denhaag-cta-download-gap) / 1.5));
+  padding-block-start: var(--denhaag-cta-download-padding-block, var(--denhaag-space-xl));
```

Comment:
Dit mag je van mij ook wel reduceren naar `padding-block` of `padding-inline` als de `-start` en `-end` gelijk zijn aan elkaar. Scheelt ook weer wat behoorlijk CSS regels in the end ;-)